src/borg/repository.py

```diff
@@ -565,3 +566,8 @@ def store_delete(self, name, *, deleted=False):
     def store_move(self, name, new_name=None, *, delete=False, undelete=False, deleted=False):
         self._lock_refresh()
         return self.store.move(name, new_name, delete=delete, undelete=undelete, deleted=deleted)
+
+def _local_abspath_to_file_url(path: str) -> str:
+    if sys.platform in ("win32", "msys", "cygwin"):
+        path = "/" + path.replace("\\", "/")
```

Comment:
why do you prepend a slash?

Reply:
See https://github.com/borgbackup/borg/pull/9097#discussion_r2462233737

Analogous issues exist for POSIX. If a caller calls it with an invalid POSIX path, like `///foo`, it'll output an "invalid" file url `file://///foo`.

Reply:
well, but \Windows\System32 is a valid windows path.

Reply:
So, maybe put a comment below that "if", that the code expects a full path including drive letter.